department-of-veterans-affairs/caseflow
# Allow global admin to stay logged in after impersonated user logout

Resolves #10035 


### Testing Plan
Prerequisite: 
Add `Functions.grant!("Global Admin", users: ["FAKE USER"])` to development.rb

1. Go to /test/users
2. Use global admin feature to log in as a different user (i.e. CSS_ID1, 101)
3. Make sure you are logged in as the new user 
4. Make sure you are on Help page 
5. Logout
6. Make sure you are logged in as previous user
7. Make sure you are on /test/users page

Local testing: 

![GlobalAdmin](https://user-images.githubusercontent.com/4960030/54753845-6d389080-4bb8-11e9-9ce0-48519e4b1044.gif)

UAT testing: 

![GlobalAdmin (1)](https://user-images.githubusercontent.com/4960030/54781377-8100e880-4bf2-11e9-993c-a7c44b024d14.gif)
